package.json

```diff
@@ -21,14 +21,16 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-games#readme",
   "dependencies": {
-    "pg-format": "^1.0.4",
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "express": "^4.18.1",
```

Comment:
adding express should not have been grouped with this ticket - and if it is your commit message should certainly make mention of it. 
Be wary of grouping things I suspect you may have used `git add .` rather than selecting these files individually... 🤔